docs/src/components/ComponentDoc/ComponentDocTag.tsx

```diff
@@ -1,27 +1,84 @@
 import * as React from 'react'
 import * as _ from 'lodash'
 import { Header, Message } from 'semantic-ui-react'
+const behaviorMenuItems = require('docs/src/behaviorMenu')
 
 const headerStyle = {
   whiteSpace: 'pre-line',
 }
 
+type behaviorFiltered = {
+  type: string
+  displayName: string
+  defaultFile: string
+}
+
 class ComponentDocTag extends React.Component<any, any> {
   getTagDescription = (forTag, fromInfo) => {
     const tags = (fromInfo.docblock && fromInfo.docblock.tags) || []
     return _.result(_.find(tags, 'title', forTag), 'description')
   }
 
+  getBehaviorURLandName = fromInfo => {
+    const defaultBeharior = {
```

Comment:
typo

Reply:
where?

Reply:
defaultBeharior -> defaultBehavior

Reply:
Remove `defaultBehavior` completely. Instead of returning `{name: undefined, url: undefined}` return just `undefined`.